### PR TITLE
Authenticate user after created

### PR DESCRIPTION
**Pull request by taylorrf**
Login after successful create a new user.

Closing card:
https://trello.com/c/zrrLKNkD/31-login-users-after-create-an-account

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def create
     user = User.new user_params
 
     if user.save
+      session.update user_id: user.id
```
Comment (andyw8) [inline app/controllers/registrations_controller.rb]: Let's add a test to verify this, e.g.

```
    it "signs the user in" do
      user = stub_user_creation_with method: :save, return_value: true

      post :create, registration: { email: "" }

      expect(session[:user_id]).to eq(user.id)
    end
```

**Comment by andyw8** [comment]
:+1: other than my comment above.

File: app/controllers/registrations_controller.rb (hunk 1 above)
Comment (taylorrf) [inline app/controllers/registrations_controller.rb]: Awesome, I'll add it :+1:

File: spec/controllers/registrations_controller_spec.rb [hunk 2]
```diff
@@ -17,6 +17,14 @@
 
       expect(response).to render_template :new
     end
+
+    it "signs the user in" do
+       user = stub_user_creation_with method: :save, return_value: true
```
Comment (taylorrf) [inline spec/controllers/registrations_controller_spec.rb]: Good catch Hound

**Pull request closed without merging**